modules/appnexusAstBidAdapter.md

```diff
@@ -0,0 +1,102 @@
+# Overview
+
+```
+Module Name:  AppnexusAst Bid Adapter
+Module Type:  Bidder Adapter
+Maintainer: info@prebid.org
+```
+
+# Description
+
+Connects to Appnexus exchange for bids.
+
+AppnexusAst bid adapter supports Banner, Video (instream and outstream) and Native.
+
+# Test Parameters
+```
+var adUnits = [
+   // Banner adUnit
+   {
+       code: 'banner-div',
+       sizes: [[300, 250], [300,600]],
+       bids: [{
+         bidder: 'appnexusAst',
+         params: {
+           placementId: '10433394'
+         }
+       }]
+   },
+   // Native adUnit
+   {
+      code: 'native-div',
+      sizes: [[300, 250], [300,600]],
+      mediaType: "native",
```

Comment:
this should use the new `mediaTypes` attribute.

Reply:
Done